### DISCUSSION TITLE
What is "chinese localization" and "russian localization"

### DISCUSSION THREAD

**Discussion by ThePolskiPolak** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1018] [Q&A] [answered]
I've noticed that they often appear in the changelogs, but I never understood what they were.

**Comment by BallOfEnergy1** [comment, marked as answer]
Localizations are what allow the mod to support different languages. In this case, russian and chinese (in mod) are being updated to fix inaccuracies in the languages.
